src/developer/nodes-system/node_type_decorators.rst

```diff
@@ -0,0 +1,102 @@
+.. _node_type_decorators:
+
+
+Node-type Decorators
+====================
+
+Roadiz allows you to customize any non-structural properties of node types and their fields.
```

Comment:
```suggestion
Roadiz allows you to customize any node-types and node-type fields **non-structural** properties. This can be handy if you need to change node-types appearence (display names, colors) in Back-office without needing to update your project configuration or emptying caches.
```

Reply:
Documentation is easier to read if you take examples. Or provide context